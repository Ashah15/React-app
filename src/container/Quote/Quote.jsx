/* eslint-disable react/react-in-jsx-scope */
import QUOTES from './Quote-data';
import './Quote.style.css';

const Quote = () => {
  const [quote, author] = QUOTES[Math.floor(Math.random() * (QUOTES.length))].split('â€“');
  return (
    <blockquote id="quote">
      {quote}
      <span>{`- ${author}`}</span>
    </blockquote>
  );
};

export default Quote;
