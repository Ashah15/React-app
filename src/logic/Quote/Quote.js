import QUOTES from './Quotes-data';
import './QuoteStyles.css';

const Quote = () => {
  const [quote, author] = QUOTES[Math.floor(Math.random() * (QUOTES.length))].split('â€“');
  return (
    <blockquote id="quote">
      {quote}
      <span>{`- ${author}`}</span>
    </blockquote>
  );
};

export default Quote;