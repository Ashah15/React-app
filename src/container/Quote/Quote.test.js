/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable react/react-in-jsx-scope */
import renderer from 'react-test-renderer';
import Quote from './Quote';

test('renders learn react link', () => {
  const quoteData = ['Test Quote -Testie'];
  const result = renderer.create(<Quote quoteData={quoteData} />).toJSON();
  expect(result).toMatchSnapshot();
});
