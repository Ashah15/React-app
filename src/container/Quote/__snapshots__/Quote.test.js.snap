// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<blockquote
  id="quote"
>
  The only way to learn mathematics is to do mathematics. 
  <span>
    - Paul Halmos
  </span>
</blockquote>
`;
